### PR TITLE
staticpages

### PR DESCRIPTION
**Pull request by tenyiting**
ready

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/static_pages.scss:1]: delete this file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (huongnt-2545) [inline app/helpers/static_pages_helper.rb:1]: delete this file, it's not useful

File: test/controllers/static_pages_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/static_pages_controller_test.rb:1]: delete this file

**Comment by tenyiting** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**